trlx/utils/modeling.py

```diff
@@ -489,3 +489,86 @@ def range_to_pattern(start, stop):
         negative_only_subpatterns + intersected_subpatterns + positive_only_subpatterns
     )
     return "|".join(subpatterns)
+
+
+def best_of_n_sampling(
```

Comment:
Why are we adding `best_of_n_sampling` here if unused in the OpenAI Summarize,  example?  Do we want to squeeze a new feature in here? CC @LouisCastricato

Reply:
I wanted to add it in @reciprocated's generation utils pr but it didn't end up happening

Reply:
It was added here on tuesday and I thought Duy had planned to make use of it. I can host it on my pr instead